### ISSUE TITLE
having the  X-axis labels all on the same line

### ISSUE THREAD

**Issue by MSaeedNajafi** [Abhinandan-Kushwaha/react-native-gifted-charts#678]
please take a look at this:
<img width="307" alt="image" src="https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/19430099/6c5b9c14-a51c-479d-add3-0cc533012268">
how can I move the name of the months all at the bottom and on the same line?
I have tried some solutions but it doesn't work, if I set the `rotateLabel` it moves them all to the center and on the same line. but I haven't been able to move them down.

```
<BarChart
            style={{
              flex: 1,
              // paddingBottom: 20,
            }}
            width={290}
            height={90}
            barWidth={10}
            spacing={selectedOption === "year" ? 20 : 10}
            xAxisLabelTextStyle={{
              fontSize: 8,
              fontFamily: "Goga-Bold",
              color: "black",
              marginBottom: -20,
            }}
            yAxisTextStyle={{
              left: 0,
              fontSize: 8,
              color: "#FF6B00",
              fontFamily: "Goga-Bold",
            }}
            stackData={displayOnGraph(data, selectedOption)}
            yAxisThickness={0}
            xAxisThickness={1}
            noOfSections={5}
            rulesThickness={0}
            formatYLabel={(label) => `€${label}`}
          />
```

**Comment by Abhinandan-Kushwaha** [comment]
Hi @MSaeedNajafi 👋
Thanks for requesting this feature.

I have added 2 new props named `autoShiftLabelsForNegativeStacks` and `labelsDistanceFromXaxis`
By default the value of `autoShiftLabelsForNegativeStacks` is `true`, if you want all the X-axis labels to be aligned in one single line, you need to set it to `false`.

Then you can use the prop `labelsDistanceFromXaxis` to shift the labels vertically up or down (perhaps you want to shift it down).

This feature is available from version `1.4.14` onwards 🎉
Please use the latest version of the library.

**Comment by MSaeedNajafi** [comment]
Thanks a lot for your help and support.
